associate.js

```diff
@@ -0,0 +1,53 @@
+var associateMap = {};
+
+exports.help = function() {
+	return '/associate "<hook>" "<text>" : Associate and disassociate a phrase with another.';
+};
+
+exports.match = function(text, thread, api) {
+	if (text.startsWith('/associate')) {
```

Comment:
I'm going to add them in server.js cause the number of times they will be useful is high